docs/whatsnew/0.3.rst

```diff
@@ -4,6 +4,5 @@
 What's New in Astropy 0.3?
 **************************
 
-See this page in the `Astropy v0.3 documentation`__.
+``See this page in the `Astropy v0.3 documentation`__.``
```

Comment:
```suggestion
See this page in the ``Astropy v0.3 documentation``.
```

Reply:
I have addressed this in the most recent commit.